testsuite/MathMLToDisplay/General/Colors/issue1002.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>Test SVG output inherting color</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../../header.js"></script>
+    <script type="text/javascript" src="../../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+      var path = document.querySelectorAll('g')[0];
+      var check1 = (path.getAttribute('fill') === 'currentColor' & path.getAttribute('stroke') === 'currentColor');
```

Comment:
You probably want `&&` rather than `&` here.  The `&&` is logical and, while `&` is bitwise and.  Both work in this case, but best to be in the habit of using `&&` and avoid bitwise and.